{"author":{"id":"924068365fc4e30a510f97409bbeb4d89b6fb4b9135adc5ed6bd23a3a584748b"},"ops":[{"type":6,"timestamp":1562424375,"nonce":"T3tzBbKmU/NxNjt9lXr1kZjgAN4=","metadata":{"github-id":"MDE1OlVzZXJDb250ZW50RWRpdDI0NTc1Mjk0NA=="},"target":"b67d4187970cd216678f3b1483a873ccdeab470cae70ef384db3ec0910564356","message":"When exporting new events to Github, creating and retrieving bugs using the graphql API can lead to some unexpected behaviour (only in some special cases):\n- creating a label `bug` then trying to create a label `Bug` will return an error `Name has already been taken`\n- having only one label named `bug` and trying to retrieve a label with name `Bug` will return a null ID.\n\nFor now to avoid this problem, try to not use two labels with the same letters like `core` and `CorE`.\n\nI wrote a message to the Github support, i'll keep this issue up to date once i have a response.","files":null},{"type":5,"timestamp":1562456594,"nonce":"sK8/nys1Fdr2Z/KaF/LN+3ox0vA=","metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDI0NjQxNDM0MDI="},"added":["area/bridge"],"removed":[]},{"type":5,"timestamp":1562456594,"nonce":"f8tsT57+eiIGb+LCLXdvP1CO0WU=","metadata":{"github-id":"MDEyOkxhYmVsZWRFdmVudDI0NjQxNDM0MDM="},"added":["kind/bug"],"removed":[]},{"type":4,"timestamp":1566169350,"nonce":"r2Tjb2iCQyglr6lwKsj6rMTxtFY=","metadata":{"github-id":"MDExOkNsb3NlZEV2ZW50MjU2NDcyMTE3Mg=="},"status":2}]}